dags/job_title_sample.py

```diff
@@ -0,0 +1,76 @@
+import logging
+import os
+from datetime import datetime
+
+from airflow import DAG
+from airflow.hooks import S3Hook
+from airflow.operators import BaseOperator
+
+from utils.airflow import datetime_to_quarter
+from utils.s3 import upload
+from utils.fs import check_create_folder
+from algorithms.jobtitle_sampler import sampler
+from config import config
+
+default_args = {
+    'depends_on_past': False,
+    'start_date': datetime(2011, 1, 1),
+}
+
+dag = DAG(
+    'jobtitle_sampler',
+    schedule_interval='0 0 1 */3 *',
+    default_args=default_args
+)
+
+output_folder = config.get('output_folder', 'output')
+if not os.path.isdir(output_folder):
+    os.mkdir(output_folder)
+
+SAMPLENUM = 100
+
+class JobTitleSampleOperator(BaseOperator):
+    def execute(self, context):
+        s3_conn = S3Hook().get_conn()
+        quarter = datetime_to_quarter(context['execution_date'])
+
+        cleaned_count_filename = '{}/cleaned_geo_title_count/{}.csv'.format(
```

Comment:
The filenames are based on the output of a prior step produced in this repo: instead of hardcoding 'cleaned_geo_title_count', can we sync them both using entries in the config module?